app/components/primer/alpha/nav_list/item.html.erb

```diff
@@ -1,7 +1,7 @@
 <% with_private_content do %>
   <% unless items.empty? %>
     <% capture do %>
-      <%= render(Primer::BaseComponent.new(tag: :ul, **@sub_list_arguments)) do %>
+      <%= render(Primer::BaseComponent.new(tag: :ul, role: :list, **@sub_list_arguments)) do %>
```

Comment:
Curious why we added the `role="list"` since the tag is a `<ul>`?

Reply:
Good question! I also wondered about this. Here's the relevant part of @jscholes' [comment](https://github.com/github/primer/issues/1120#issuecomment-1279387613) explaining:

> The `<ul>` wrapping the items in the "Moderation options" submenu does not have an accessible name. It also has no role of `"list"`, meaning that WebKit-based browsers will strip the list semantics due to the lack of bullets.
>
>Recommendations: add `role="list"` to the `<ul>`, as well as an `aria-labelledby` attribute referencing the ID of the "Moderation options" button.

Reply:
Love it, thank you!